src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -876,6 +888,10 @@ private void CheckModifiers(bool isExplicitInterfaceImplementation, Location loc
                 // '{0}' cannot be sealed because it is not an override
                 diagnostics.Add(ErrorCode.ERR_SealedNonOverride, location, this);
             }
+            else if (IsPartial && IsAbstract)
+            {
+                diagnostics.Add(ErrorCode.ERR_PartialMemberCannotBeAbstract, location);
```

Comment:
It's not great that we have to hunt down through 3 specs (partial properties, extended partial methods, base specs) to figure out the rules... Consider making the partial properties spec more standalone.

Along those lines, it looks like partial methods disallow explicit interface implementations (`ERR_PartialMethodNotExplicit`). The same should apply to partial properties.
Also, partial properties should only appear in partial types (analog to `ERR_PartialMethodOnlyInPartialClass`).

Reply:
I reviewed the feature spec [partial-properties.md](https://github.com/dotnet/csharplang/blob/main/proposals/partial-properties.md) in search of what rules were missing from the spec itself and require looking at other documents.

I wasn't able to find any rules missing in this check, but it's quite possible I missed something.

[Defining and implementing declarations](https://github.com/dotnet/csharplang/blob/main/proposals/partial-properties.md#defining-and-implementing-declarations) states the following. 
> A partial property is not permitted to have the `abstract` modifier.

It is at the bottom of the section, and perhaps easy to miss, which might mean that the spacing in the doc should be adjusted to emphasize which parts are actual rules.

Reply:
Thanks! Sorry I'd missed it.
What's the plan for explicit interface implementations? For methods, we have "A partial method cannot explicitly implement interface methods."

Reply:
I'll update the feature spec to block partial explicit implementation of interface properties.

Reply:
dotnet/csharplang#8102

Reply:
I added errors for the explicit implementation and non-partial containing type cases. I ctrl+F'd through ErrorCode.cs for "partial" and I think that all the partial errors relevant for this PR are covered. (errors related to indexer parameters are not covered, those are for a future PR.)